src/main/scala/DataMunging.scala

```diff
@@ -0,0 +1,40 @@
+/**
+  * Created by idarlington on 2/26/17.
+  */
+object DataMunging {
+
+  case class weatherLine(
+                          dayNumber : Int,
+                          maxTemperature : Double,
+                          minTemperature : Double
+                        )
+
+  def temperatureDiff (a : weatherLine, b : weatherLine) = {
+    (a.maxTemperature - a.minTemperature) > (b.maxTemperature - b.minTemperature)
+  }
+
+  def main(args: Array[String]) = {
```

Comment:
I suppose you could extend App instead of having a main method, but thats just a style thing. Whichever style you like is good.